subsys/nfc/lib/platform.c

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (c) 2018 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+#include <clock_control.h>
+#include <device.h>
```

Comment:
Not needed. You'll get this via `<drivers/clock_control.h>`.

Reply:
Ok